src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,91 @@
+package duke.task;
+
+<<<<<<< HEAD
+import java.time.LocalDate;
+
+/**
+ * An abstract class representing the Tasks included in Duke.
+ */
+=======
+>>>>>>> branch-A-CodingStandard
```

Comment:
I think you forgot to delete these lines again when resolving your merge conflict?

```suggestion
import java.time.LocalDate;

/**
 * An abstract class representing the Tasks included in Duke.
 */
```